src/cljs/nr/account.cljs

```diff
@@ -478,27 +539,8 @@
                    :on-click #(clear-card-art s)}
                   (tr [:settings.reset "Reset All to Official Art"])])]])]
 
-         [:section
-          [:h3 (tr [:settings.blocked "Blocked users"])]
-          [:div
-           [:input {:on-change #(swap! s assoc-in [:block-user-input] (-> % .-target .-value))
-                    :on-key-down (fn [e]
-                                   (when (= e.keyCode 13)
-                                     (.preventDefault e)
-                                     (add-user-to-block-list user s)))
-                    :ref "block-user-input"
-                    :value (:block-user-input @s)
-                    :type "text" :placeholder (tr [:settings.user-name "User name"])}]
-           [:button.block-user-btn {:type "button"
-                                    :name "block-user-button"
-                                    :on-click #(add-user-to-block-list user s)}
-            (tr [:settings.block "Block user"])]]
-          (doall (for [bu (:blocked-users @s)]
-                   [:div.line {:key bu}
-                    [:button.small.unblock-user {:type "button"
-                                                 :on-click #(remove-user-from-block-list % s)} "X" ]
-                    [:span.blocked-user-name (str "  " bu)]]))]
-
+         (blocked-users user s)
```

Comment:
Use brackets so they're handled by reagent's reactive rendering logic.

Reply:
TBH I don't really know much about reagent lol

Reply:
You should at least read this page to get a sense of what's possible: http://reagent-project.github.io/